example/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_executable(inexor-vulkan-renderer-example
+    main.cpp)
+
+set_target_properties(inexor-vulkan-renderer-example PROPERTIES
+    CXX_EXTENSIONS OFF
```

Comment:
Why off?

Reply:
What extensions do we use? Using non standard things is probably bad.

Reply:
https://cmake.org/cmake/help/latest/prop_tgt/CXX_EXTENSIONS.html not sure if thats what it does.